frontend/src/app/utils/func-utils.js

```diff
@@ -9,6 +9,22 @@ const handlerFuncNameRegExp = /^[a-zA-Z_$][a-zA-Z0-9_$]*$/;
 export const funcSizeLimit = unitsInBytes.MEGABYTE * 100;
 export const handlerFileSuffix = '.js';
 export const memorySizeOptions = deepFreeze([
+    {
+        value: 8,
+        label: '8 MB'
+    },
+    {
+        value: 16,
+        label: '16 MB'
+    },
+    {
+        value: 32,
+        label: '32 MB'
+    },
+    {
+        value: 64,
+        label: '64 MB'
+    },
```

Comment:
Currently, the code just uses the first option as the default value. This change will make the default value (in the selection box) 8MB. I think we would like that because most users just use the default value

A good fix is to add a boolean field to the default value:
```
{
        value: 128,
        label: '128 MB'	        
        default: true
}
```

Then change line 144 in `create-func-modal.js` from:
```
        handlerFunc: '',
        memorySize: memorySizeOptions[0].value,
        timeoutMinutes: 0,
```
to: 
```
        handlerFunc: '',
        memorySize: memorySizeOptions.find(opt => opt.default).value,
        timeoutMinutes: 0,
```